README.md

```diff
@@ -1,7 +1,9 @@
 # Office-365-Connector
 Office 365 Connector plugin for Jenkins
 
-Started with a fork of the [Workplace plugin](https://github.com/Outconn2016/workplace-plugin)
+Plugin is used to send actionable messages in Outlook, Office 365 Groups, and Microsoft Teams.
```

Comment:
This is small but very important change because we should also maintain public relation of this tool as the chart from https://wiki.jenkins.io/display/JENKINS/Office+365+Connector+Plugin proves that this is going to be more and more popular.

So then I would suggest to provides links to mentioned software so Google  ranks this page higher.

Reply:
So a link for outlook, office 365 groups and teams?
If I understood you correctly 👍

Reply:
True. Just for Google rank